pkg/ddc/alluxio/operations/base.go

```diff
@@ -293,6 +295,28 @@ func (a AlluxioFileUtils) IsMounted(alluxioPath string) (mounted bool, err error
 	return mounted, err
 }
 
+func (a AlluxioFileUtils) UnmountedAlluxioPath(alluxioPaths []string) (unmountedAlluxioPath []string, err error) {
```

Comment:
According to my understand, the purpose of this method is to find unmounted Alluxio path? How about name it `findUnmountedAlluxioPaths` ?

Reply:
Fixed as suggested